app/services/permission.rb

```diff
@@ -42,6 +42,7 @@ def registered_user_permissions
     return true if controller == "passwords" && action.in?(%w(edit update))
     return true if controller == "upvotes" && action.in?(%w(create destroy))
     return true if controller == "downvotes" && action.in?(%w(create destroy))
+    return true if controller == "best_answers" && action.in?(%w(create))
```

Comment:
I'm glad you had the idea to lean out these permissions methods to keep it DRYer in here.